### ISSUE TITLE
Failed to render "editURL"

### ISSUE THREAD

**Issue by keyolk** [matcornic/hugo-theme-learn#29]
the editURL is not rendered properly

below is my config.toml
```
[params]
  description = "hugo contents for keyolk.github.io"
  author = "keyolk"
  showVisitedLinks = true
  search = true
  editURL = "https://github.com/keyolk/keyolk.github.io-hugo/edit/master/content/"
  menu = [ "workspace", "container", "system", "security", "language", "algorithm" ]
```

It works after I change below 
```
diff --git a/layouts/partials/header.html b/layouts/partials/header.html
index 030139c..1f91f71 100644
--- a/layouts/partials/header.html
+++ b/layouts/partials/header.html
@@ -29,12 +29,12 @@
         <div class="padding highlightable">

             <div id="top-bar">
-              {{ if and .IsPage .Site.Params.editURL }}
+              {{ if and .IsPage .Site.Params.editurl }}
                 {{ $File := .File }}
                 {{ $Site := .Site }}
                 {{with $File.Path }}
               <div id="top-github-link">
-                  <a class="github-link" href="{{ $Site.Params.editURL }}{{ replace $File.Dir "\\" "/" }}{{ $File.LogicalName }}" target="blank">
+                  <a class="github-link" href="{{ $Site.Params.editurl }}{{ replace $File.Dir "\\" "/" }}{{ $File.LogicalName }}" target="blank">
                     <i class="fa fa-code-fork"></i>
                     Edit this page
                   </a>
```
I'm not sure, it seems the pram names is used with lower case when it templated.

```bash
$ hugo version
Hugo Static Site Generator v0.17 BuildDate: 2016-12-18T03:13:22-08:00
```

**Comment by matcornic** [comment]
Hello @keyolk,

Actually, it seems to be same the problem on the other issue, referencing : https://github.com/tummychow/lanyon-hugo/issues/2#issuecomment-253274278 

Original issue is : https://github.com/spf13/hugo/issues/2740

It should be fixed in 0.18 release. So, update to 0.18  and try again :smiley:

**Comment by keyolk** [comment]
Thanks to quick reply.
I confirmed it is solved.
